### DISCUSSION TITLE
Page instead of post as front page

### DISCUSSION THREAD

**Discussion by charlesgoforth** [BoidCMS/BoidCMS#14] [Ideas]
I would like to use the CMS's page creation instead of post/blog creation and have the front page not display blog posts. Is there a way to have page content on the front of the site instead of posts? Thanks.

**Comment by sysadev** [comment]
Hello @charlesgoforth 
[Here is a plugin](https://github.com/BoidCMS/custom-home) which can help you about that.

**Comment by BoidCMS** [comment]
Hey @charlesgoforth, This feature has been added in the new release [`v2.0.0`](https://github.com/BoidCMS/BoidCMS/releases/tag/v2.0.0)
